src/db/migrations/20210119205729-create-pet.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable("Pets", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      name: {
+        type: Sequelize.STRING,
+      },
+      age: {
+        type: Sequelize.INTEGER,
+      },
+      weight: {
+        type: Sequelize.INTEGER,
+      },
+      gender: {
+        type: Sequelize.STRING,
+      },
+      breed: {
+        type: Sequelize.STRING,
+      },
+      storeId: {
+        type: Sequelize.INTEGER,
+      },
+      ownerId: {
```

Comment:
you will need to define the relationships you have established in the models inside the migration files as well. here's an example.
<img width="492" alt="Screen Shot 2021-01-19 at 5 46 49 PM" src="https://user-images.githubusercontent.com/31671880/105107724-4f6f3780-5a7e-11eb-8f45-d3b3876cd947.png">

Reply:
Once your on a project, adding a migration file to change/update stuff is best practice as it leaves a paper trail of all modification to the db.

Reply:
Also, sometimes you'll try to build a relationship w/ a foreign key referencing a table that has yet to be migrated. Your first thought might be to reorder the migration file, but don't do it! It's always better to add  a new migration file and with the new relationship after both tables have been created.